SwiftSemanticSearch/SearchModel.swift

```diff
@@ -180,6 +180,7 @@ class SearchModel: ObservableObject {
         do {
             // Get the embedding for the query.
             let queryEmbedding = try textEncoder.encode(query).asFloats()
+            imageIndex.reserve(100)
```

Comment:
Hard-coding these values is a bad idea. Also doesn't make sense to call that function here. 

This should be patched inside of USearch ObjC layer.